lib/pacemaker/pcmk_graph_producer.c

```diff
@@ -449,12 +449,12 @@ create_graph_action(xmlNode *parent, pcmk_action_t *action, bool skip_details,
             interval_ms = 0;
         }
         clone_key = clone_op_key(action, interval_ms);
-        crm_xml_add(action_xml, XML_LRM_ATTR_TASK_KEY, clone_key);
-        crm_xml_add(action_xml, "internal_" XML_LRM_ATTR_TASK_KEY,
+        crm_xml_add(action_xml, PCMK__XA_OPERATION_KEY, clone_key);
+        crm_xml_add(action_xml, "internal_" PCMK__XA_OPERATION_KEY,
```

Comment:
I don't see where anything uses "internal_operation_key" but I'm afraid to remove it (e.g., resource definition change?).

Reply:
Doesn't look like it's used anywhere, but it's worth having in the graph for simulation debugging purposes